.github/workflows/release.yml

```diff
@@ -12,19 +12,27 @@ jobs:
         operating-system: [ubuntu-latest]
         php-versions: ['7.3']
```

Comment:
Obsolete?

Reply:
The idea was to leave the matrix in case we need it in the future to add new version